_docs_operate/configuration.md

```diff
@@ -595,7 +595,47 @@ This module is deprecated in favor of the [Message Broker Publisher](#messagebro
 
   The server under the URL must respond to the request with a status code between 200 and 299. Otherwise the Connector will log a warning.
 
-  <br>
+  Additionally, you can configure an `authenticationProvider` for a target. The authentication provider is used to authenticate the request to the webhook. The available authentication providers are: `OAuth2` and `ApiKey`.
+
+  **OAuth2**
+
+  The OAuth2 authentication provider is used to authenticate the request to the webhook using OAuth2. The Connector will send a bearer token as part of the request in its Authentication header. To configure the OAuth2 authentication provider, you need to provide the following parameters:
+
+  - **type** `"OAuth2", required`
```

Comment:
```suggestion
  - **type** `"OAuth2" | "ApiKey", required`
```

Reply:
nope, this only counts for the OAuth2 part and therefore HAS to be OAuth2